spec/money_tracker_spec.rb

```diff
@@ -0,0 +1,44 @@
+describe MoneyTracker do
```

Comment:
Would it be possible for the `MoneyTracker` tests to use doubles for the collaborator classes?